Woof/Woof/ViewModel/DetailPetSitterViewModel.swift

```diff
@@ -17,6 +18,14 @@ final class DetailSitterViewModel: ObservableObject {
     /// The URL of the sitter's avatar image.
     @Published var imageURL: URL?
 
+    /// Copies the text to the clipboard. The copied text is available in the
+    /// systemwide general clipboard, which you use for general copy-paste operations.
+    ///
+    /// - Parameter text: The text to be copied to the clipboard.
+    func copyToClipboardText(_ text: String) {
+        UIPasteboard.general.string = text
+    }
+
```

Comment:
nit:
Can we place this method under init, it's strange for me to see the initializer at the end?